sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -25,7 +25,7 @@ import kotlin.math.pow
  */
 internal class MapboxRenderThread : Choreographer.FrameCallback {
 
-  internal val handlerThread: WorkerHandlerThread
+  internal val handlerThread: RenderHandlerThread
```

Comment:
maybe choose one name for instance of this object?
I found three different names: `handlerThread`, `renderThread`, `renderHandlerThread`

I vote for `renderHandlerThread`

Reply:
Done